pkg/kudo/operator.go

```diff
@@ -205,15 +222,74 @@ func (operator Operator) Uninstall() error {
 		Delete(operator.Operator.Name, &options)
 	if err != nil {
 		return fmt.Errorf(
-			"failed to delete Operator %s in namespace %s: %w",
+			"failed to delete Operator %s in namespace %s: %v",
 			operator.Operator.Name,
 			operator.Operator.Namespace,
 			err)
 	}
 
+	if timeout != 0 {
+		kudoV1beta1 := operator.client.Kudo.KudoV1beta1()
+		i := operator.Instance
+		ov := operator.OperatorVersion
+		o := operator.Operator
+		timeoutSeconds := timeout.Round(time.Second).Milliseconds() / 1000
```

Comment:
Nit: Hmm, or just cast `timeout.Seconds()` to `int64`? Not tested :)
```suggestion
		timeoutSeconds := int64(timeout.Seconds())
```

Reply:
I'd rather keep the rounding semantics: https://play.golang.org/p/aPkzRuYuxL6